lib/renderer/callbacks-registry.ts

```diff
@@ -23,14 +24,14 @@ class CallbacksRegistry {
     let filenameAndLine
     let match
 
-    while ((match = regexp.exec(stackString)) !== null) {
+    while ((match = regexp.exec(stackString!)) !== null) {
```

Comment:
Can we do a falsey check above here to ensure that it isn't null, better safe than telling TS we know better.